lib/primitives/base-element.js

```diff
@@ -116,10 +137,22 @@ module.exports = function(registry) {
     },
 
     fromRefract: function(doc) {
-      this.meta = doc.meta;
       this.attributes = doc.attributes;
       this.content = doc.content;
 
+      if (doc.meta) {
+        Object.keys(doc.meta).forEach(function (key) {
+          var value = doc.meta[key];
+
+          // TODO: Warning - this may raise a false positive :-(
```

Comment:
After our chat today I've updated this to only check `element`, since that is all that is required according to the spec. Also updated the README with info.